pkg/apis/newrelic/v1alpha1/monitor_types.go

```diff
@@ -100,6 +101,11 @@ func (s *Monitor) IsCreated() bool {
 	return s.Status.IsCreated()
 }
 
+// Signature for the CRD
+func (s *Monitor) Signature() string {
```

Comment:
any reason you added this back?

Reply:
Did not know about the context logger. Removed.